pdns/distributor.hh

```diff
@@ -98,6 +98,8 @@ public:
   {
     QuestionData(const Question& query): Q(query)
     {
+       id = 0;
```

Comment:
I prefer {} initializers with the fields themselves: nicething is anybody adds a ct and forget the init, things will keep on working.